athenz-identity-provider-service/src/main/java/com/yahoo/vespa/hosted/athenz/instanceproviderservice/AthenzSslKeyStoreConfigurator.java

```diff
@@ -82,35 +124,41 @@ public void deconstruct() {
     }
 
     Instant getKeyStoreExpiry() throws KeyStoreException {
-        X509Certificate certificate = (X509Certificate) currentKeyStore.getCertificate(CERTIFICATE_ALIAS);
-        return certificate.getNotAfter().toInstant();
+        return getKeyStoreExpiry(currentKeyStore);
     }
 
+    private static Instant getKeyStoreExpiry(KeyStore keyStore) throws KeyStoreException {
```

Comment:
Unprecise name, the keystore doesn't have a expiry, the certificate does...

Reply:
Will fix.